packages/fxa-content-server/app/scripts/templates/support.mustache

```diff
@@ -1,12 +1,14 @@
 <div id="fxa-settings-header-wrapper">
   <header id="fxa-settings-header">
-    <h1 id="fxa-manage-account"><span class="fxa-account-title">{{#t}}Firefox Accounts{{/t}}</span></h1>
+    <h1 id="fxa-manage-account" class="text-xl">
+      <span class="fxa-account-title font-bold ltr:mr-2 rtl:ml-2">{{#t}}Firefox Accounts{{/t}}</span>
+    </h1>
   </header>
 
   <ol class="breadcrumbs">
-    <li><a href="/settings">{{#t}}Account Home{{/t}}</a></li>
-    <li><a href="/subscriptions">{{#t}}Subscriptions{{/t}}</a></li>
-    <li><a href="/support">{{#t}}Contact Us{{/t}}</a></li>
+    <li><a class="text-[#0060df] no-underline" href="/settings">{{#t}}Account Home{{/t}}</a></li>
+    <li><a class="text-[#0060df] no-underline" href="/subscriptions">{{#t}}Subscriptions{{/t}}</a></li>
+    <li><a class="text-[#0060df] no-underline" href="/support">{{#t}}Contact Us{{/t}}</a></li>
```

Comment:
Looks like we're using the custom `text-[#0060df]` color 5&times; in this PR (and I think only in this PR; at least per <kbd>git grep -n "#0060df" | grep -v ".svg:"</kbd> in the `main` branch).
Not sure if we want to maybe add it as a custom color in the Tailwind config theme. :shrug:

Reply:
I updated to use the color mentioned by @xlisachan.